.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      # Default is true, cancels jobs for other platforms in the matrix if one fails
+      fail-fast: false
+      matrix:
+        # The build currently fails on Windows because of Olric. See https://github.com/buraksezer/olric/pull/64.
+        #os: [ ubuntu-latest, macos-latest, windows-latest ]
```

Comment:
It's been merged, should the dependency be updated to target a specific commit, and CI turned back on?